armi/bookkeeping/report/reportInterface.py

```diff
@@ -166,11 +167,66 @@ def writeReports(self):
                 report_.writeHTML()
 
     # --------------------------------------------
-    #        Misc Summaries
+    #        Ex-Core Summaries
     # --------------------------------------------
     def writeRunSummary(self):
         """Make a summary of the run."""
         # spent fuel pool report
-        if self.r.sfp is not None:
-            self.r.sfp.report()
-            self.r.sfp.count()
+        if self.r.excore.get("sfp") is not None:
+            self.reportSFP(self.r.excore["sfp"])
+            self.countAssembliesSFP(self.r.excore["sfp"])
+
+    @staticmethod
+    def reportSFP(sfp):
```

Comment:
These are supposed to "reporting tools".

They are a heck of a lot more useful if people can mix-and-match and grab on the go then if you have to build a full `Operator` and grab the `Interface`.

And it was a trivially easy change to make. Since they only take the ONE object, and nothing else from the `self` space.

(I won't die on this hill.)

Reply:
True enough, I agree